_upgrade-to/upgrade-to.md

```diff
@@ -18,6 +18,8 @@ If your existing cluster runs an older version of Elasticsearch OSS, the first s
 
   Cluster restart upgrades work between minor versions (for example, 6.5 to 6.8) and the next major version (for example, 6.x to 7.10.2). Cluster restart upgrades are faster to perform and require fewer intermediate upgrades, but require downtime.
 
+To migrate a post-fork version of Elasticsearch (7.11+) to OpenSearch, you can use Logstash.  You'll need to employ the Elasticsearch input plugin within Logstash to extract data from the Elasticsearch cluster, and the [Logstash Output OpenSearch plugin](https://github.com/opensearch-project/logstash-output-opensearch#configuration-for-logstash-output-opensearch-plugin) to write the data to the OpenSearch 2.x cluster. We suggest using Logstash version 7.13.4 or earlier, as newer versions may encounter compatibility issues when establishing a connection with OpenSearch due to changes introduced by Elasticsearch subsequent to the fork. We strongly recommend that users test this solution with their own data to ensure effectiveness. 
```

Comment:
lgtm! Thanks @hdhalter . @kartg can you pls confirm the logstash version mentioned is the right one?